org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -964,14 +964,25 @@ private void loadPrintData (PrintData pd, MPrintFormat format)
 											valueString = DisplayType.getDateFormat(pdc.getDisplayType(), m_language).format(value);
 										if (format.getTableFormat().isPrintFunctionSymbols())		//	Translate Sum, etc.
 											valueString	+= PrintDataFunction.getFunctionSymbol(functions[f]);
+										if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f])) {
+											Object groupValue = m_group.getValue(pdc.getAD_PrintFormatItem_ID(), 
+													pdc.getAD_PrintFormatItem_ID(), functions[f]);
+											if(groupValue instanceof Timestamp)
+												groupValue = DisplayType.getDateFormat().format((Timestamp)groupValue);
```

Comment:
should be DisplayType.getDateFormat(pdc.getDisplayType()) instead, same goes for other places that has similar call.